assembl/static2/js/app/components/login/signupForm.jsx

```diff
@@ -125,4 +159,10 @@ const mapDispatchToProps = dispatch => ({
   }
 });
 
-export default connect(mapStateToProps, mapDispatchToProps)(SignupForm);
\ No newline at end of file
+const withData = graphql(TabsConditionQuery, {
+  props: ({ data }) => ({
+    hasTermsAndConditions: data.hasTermsAndConditions
```

Comment:
don't forget to pass `error` and `loading` here (here you can just add `...data`)